llpc/lower/llpcSpirvLower.cpp

```diff
@@ -154,6 +154,24 @@ void SpirvLower::removeConstantExpr(Context *context, GlobalVariable *global) {
     replaceConstWithInsts(context, constVal);
 }
 
+// =====================================================================================================================
+// Replace global variable with another global variable
+//
+// @param global : Replaced global variable
+// @param replaceGlobalglobal : Replacing global variable
+void SpirvLower::replaceGlobal(GlobalVariable *global, GlobalVariable *replaceGlobal) {
+  removeConstantExpr(m_context, global);
+  for (User *user : global->users()) {
+    Instruction *inst = dyn_cast<Instruction>(user);
+    assert(inst != nullptr);
```

Comment:
You can simplify this with `cast` instead of `dyn_cast`: https://llvm.org/docs/ProgrammersManual.html#the-isa-cast-and-dyn-cast-templates